lib/pages/incident.dart

```diff
@@ -200,4 +215,40 @@ class _IncidentPageState extends State<IncidentPage> {
       }).toList(),
     );
   }
+
+  Widget showAlertDialog(BuildContext context) {
+    // set up the buttons
+    Widget cancelButton = FlatButton(
+      child: Text("Cancel"),
+      onPressed: () {
+        Navigator.pop(context);
+      },
+    );
+    Widget deleteButton = FlatButton(
+      child: Text("Delete"),
+      onPressed: () {
+        _delete();
+        Navigator.of(context)
+            .push(MaterialPageRoute(builder: (BuildContext context) {
+          return IncidentsListPage();
+        }));
```

Comment:
This will push a new page to the stack, on going back it would go back to the incident that was removed. I think this should work
```suggestion
        Navigator.of(context).popUntil(ModalRoute.withName("/home"));
```
And then add this to the home route definition inside `main.dart`
```
MaterialPageRoute(
	settings: RouteSettings(name: "/home"), // this is the line to add
	...);
```